RetroAchievements/RAWeb
# feat(download): add events to the external links

At some point, I'd like to redesign this page. However, given that it gets a lot of traffic, we should ensure any redesign doesn't hurt the page's current activity.

Now that we have analytics (see [here](https://discord.com/channels/476211979464343552/1002696640001478769/1272694681062477877)), we can start measuring how often the buttons on this page are clicked in order to run a redesign through an A/B test later. Ideally, a redesign will either perform the same or better than the current page.

**How to test this PR:**
* Open your browser's network tab.
* Filter by Fetch/XHR.
* When clicking any of the links on the page, you should see events being emitted:

![Screenshot 2024-08-15 at 8 59 25 AM](https://github.com/user-attachments/assets/80092c32-0b84-4380-8769-93f61d5aee0c)
